roles/cloud-cloudstack/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+- block:
+    - name: Build python virtual environment
+      import_tasks: venv.yml
+
+    - block:
+      - name: Include prompts
+        import_tasks: prompts.yml
+
+      - name: Security group created
+        local_action:
```

Comment:
Do we really have to use `local_action`?

Reply:
this is a requirement from the `cs_*` modules yes. They are designed to be run from instances as I understood  it.

Reply:
The roles already executes on the localhost, I don't think we need this

Reply:
indeed, corrected. Thank you